runtime/src/accounts_db.rs

```diff
@@ -2896,33 +2925,34 @@ impl AccountsDB {
                         let entry = accum
                             .entry(stored_account.meta.pubkey)
                             .or_insert_with(Vec::new);
+                        self.accounts_index.update_secondary_indexes(
```

Comment:
well, secure snapshot isn't still a thing. But, I think this is newly susceptible to this secondary index poisoning? Or at least, unbounded memory growth?

So, we should check actual aliveness of each entries in the mint index comparing against the accounts index at last stage of `generate_index()`.

Reply:
Or we can just add TODO comment.

Also, I think we can construct mint index at last of `generate_index()` using `account_index` after the whole account hash check, maybe?

Reply:
@carllin ah, this security concern is maybe fixed now because purge_secondary_indexes_by_inner_key is correctly wired in the primary index codepath, right?

Reply:
@ryoqun yeah it should be bounded by the accounts_index now